aws_lambda_powertools/event_handler/api_gateway.py

```diff
@@ -128,16 +132,29 @@ def __init__(
         allow_credentials: bool
             A boolean value that sets the value of `Access-Control-Allow-Credentials`
         """
-        self.allow_origin = allow_origin
+        self.allowed_origins = [allow_origin]
+        if extra_origins:
+            self.allowed_origins.extend(extra_origins)
         self.allow_headers = set(self._REQUIRED_HEADERS + (allow_headers or []))
         self.expose_headers = expose_headers or []
         self.max_age = max_age
         self.allow_credentials = allow_credentials
 
-    def to_dict(self) -> Dict[str, str]:
+    def to_dict(self, origin: Optional[str]) -> Dict[str, str]:
```

Comment:
origin: str = ""?

Annotation says it's optional but we are not setting a default value. 

You can drop the Optional (None), and simply set to an empty str

Reply:
Does this mean I would have to compare `if origin == ""` later on instead of `not origin`? Doesn't it sound wrong?

Reply:
TIL `not` will eventually translate to `__bool__`, so it works for empty strings too